src/me/remie/vulcan/leaguetasks/task/tasks/fishing/FishAnchovies.java

```diff
@@ -0,0 +1,187 @@
+import me.remie.vulcan.leaguetasks.data.LeagueScriptConstants;
+import me.remie.vulcan.leaguetasks.helpers.ItemDetails;
+import me.remie.vulcan.leaguetasks.helpers.shop.ShopHelper;
+import me.remie.vulcan.leaguetasks.helpers.shop.ShopType;
+import me.remie.vulcan.leaguetasks.task.LeagueTask;
+import simple.robot.utils.WorldArea;
+import net.runelite.api.coords.WorldPoint;
+import simple.hooks.wrappers.SimpleNpc;
+
+/**
+ * Created by McRea on Mar 10, 2024
+ *
+ * @author mcrea
+ * @Discord mcrea
+ */
+public class FishAnchovies extends LeagueTask {
+
+    private final int FISHING_SPOT_REGION = 11317; // Catherby shore
+    private final ShopHelper fishingNetShopHelper;
+    private final int FISHING_NET_ID = 303;
+    private final ItemDetails SMALL_FISHING_NET =  new ItemDetails("Small fishing net", FISHING_NET_ID, 300);
+    private final int FISHING_SPOT_NPC_ID = 1518;
+    private final WorldPoint[] catherbyBankToFishingSpotPath = {
+            new WorldPoint(2808, 3439, 0),
+            new WorldPoint(2811, 3435, 0),
+            new WorldPoint(2817, 3436, 0),
+            new WorldPoint(2823, 3437, 0),
+            new WorldPoint(2829, 3437, 0),
+            new WorldPoint(2834, 3435, 0),
+            new WorldPoint(2839, 3434, 0),
+            new WorldPoint(2843, 3432, 0),
+            new WorldPoint(2847, 3430, 0),
+    };
+
+    private final WorldArea fishingSpotArea = new WorldArea(
+            new WorldPoint(2844, 3433,0),
+            new WorldPoint(2857, 3422,0)
+    );
+
+    /**
+     * This task achieves fishing level, fish for upcoming task.
+     * While completing this task we will also complete the following task:
+     * - Catch a Shrimp
+     */
+    public FishAnchovies(final LeagueScript script) {
+        super(script, "Catch an Anchovy");
+        this.fishingNetShopHelper = new ShopHelper(this);
+    }
+
+    @Override
+    public void run() {
+        // Validate we got everything before continue
+        if (!doWeHaveRequiredItem()) {
+            if (!teleportHome()) {
+                return;
+            }
+            if (!getRequiredItem()) {
+                ctx.log("Issues getting required items.");
+                return;
+            }
+            return;
+        }
+
+        // Teleport to Catherby
+        if (!areWeInCatherbyRegion()) {
+            if (!teleportHome()) {
+                return;
+            }
+            teleportToCatherby();
+            return;
+        }
+
+        // Navigate to bank and deposit inventory
+        if (ctx.inventory.inventoryFull()) {
+            depositItemsInCatherbyBank();
+            return;
+        }
+
+        // Navigate to fishing spot
+        if (!fishingSpotArea.within()) {
+            navigateToFishingSpot();
+            return;
+        }
+
+        if (fishAnchovies()) {
+            // Catch some extra fish for next task
+            ctx.onCondition(() -> !ctx.players.getLocal().isAnimating(), 1500, 15);
+        }
```

Comment:
Added this part, to get more fish before actually leaving for next task.

Reply:
> Added this part, to get more fish before actually leaving for next task. But I'm really considering to change this script itself to have cook in lum cooking range but have this task as also including task.

I agree having this script handling cooking as well would be ideal, but why not just cooking at catherby as well?

Reply:
You know what... i will change this to "Lumbridge" cooker task instead.
Reason for that is that there is a task to cook in there.
So it would include total of 4 task in a row. (40p total)